src/utils/rest-api.js

```diff
@@ -456,13 +456,11 @@ export function duplicateCase(
 }
 
 export function elementExists(directoryUuid, elementName, type) {
-    const elementNameEncoded = new URLSearchParams({
-        elementName: elementName,
-    }).toString();
-
     const existsElementUrl =
         PREFIX_DIRECTORY_SERVER_QUERIES +
-        `/v1/directories/${directoryUuid}/elements/${elementNameEncoded}/types/${type}`;
+        `/v1/directories/${directoryUuid}/elements/${encodeURIComponent(
```

Comment:
It's better to leave it in a variable as it was before

Reply:
Done